tests/provisioning-manifest-build.yml

```diff
@@ -0,0 +1,4 @@
+- installBundle:
+    - 'npm:mvn:org.jahia.modules.npm/luxe-jahia-demo/0.3.0-SNAPSHOT/tgz'
```

Comment:
shouldn't we use only stable versions here and use SNAPSHOT ones in the `provisioning-manifest-snapshot.yml` file?

Reply:
good catch. this should be removed as we should be using the CI built version and not from mvn.